content/doc/book/managing/users.adoc

```diff
@@ -1,15 +1,145 @@
+
+- Creating new users.
+- Configuring user details such as full name, email address, and password.
+- Modifying user information.
+- Deleting users.
+
+Administrators with the appropriate permissions can perform these actions.
+
+== Accessing the users page
+
+To manage users in Jenkins, you need to have administrative privileges. Follow these steps to access the user management section:
+
+. Log in to your Jenkins instance with an account that has administrative permissions.
++
+. Navigate to the **Manage Jenkins** page from the Jenkins dashboard.
++
+image::managing/manager-users-home-page.png[Accessing Manage Jenkins page,width=800]
+
+. Select **Users** under the **Security** section.
++
+image::managing/select-users.png[Accessing the Users section,width=800]
+
+== Creating a user
+
+To create a new user in Jenkins:
+
+. On the **Users** page, select **Create User**.
++
+image::managing/create-users-click.png[Accessing the Create User button,width=800]
+
+. Fill in the required details:
+   - **Username**: A unique identifier for the user.
+   - **Password**: A secure password for the user.
+   - **Confirm Password**: Re-enter the password.
+   - **Full Name**: The full name of the user.
+   - **Email Address**: The email address of the user.
++
+image::managing/create-users.png[Accessing the Create User Page,width=700]
+
+. Select **Create User** to save the new user.
++
+For example:
++
+image::managing/create-users-example.png[Create User Page Example,width=700]
+
+== Configuring user settings
+
+Once a user is created, you can configure their settings:
+
+. On the **Manage Users** page, select the user you want to configure.
++
+. Update the following details as needed:
+   - **Full Name**: Edit the user's full name.
+   - **Description**: Update the user's description.
+   - **Credentials**: Change the user's credentials.
++
+image::managing/account-preference.png[Configure page,width=800]
+
+. Select **Save** to apply the changes.
++
+image::managing/select-user.png[Configuring User page,width=800]
+
+== Modifying user information
+
+Administrators can modify user information at any time:
+
+. Navigate to the **Manage Users** page.
++
+. Select the user whose information you want to modify.
++
+. Update the relevant fields such as full name, email address, or password.
++
+. Select **Save** to apply the changes.
+
+== Deleting a user
+
+To remove a user from Jenkins, there are two approaches. Let’s explore both of them:
+
+=== First approach
+
+. Navigate to **Users**.
++
+. Find the user you want to delete.
++
+. Click on the **Delete** button next to the user’s name.
++
+image::managing/delete-users.png[Deleting User,width=700]
++
+
+. Confirm the deletion.
+
+=== Second approach
+
+. Navigate to **Manage Jenkins**.
++
+. Then navigate to **Users**.
++
+. Select the trash icon on the right-most side of the user you want to delete.
++
+image::managing/delete-user-alternative.png[Deleting User Alternative,width=800]
```

Comment:
This image needs to be replaced with one that is freshly captured. The red box from the previous version of this screenshot is still visible and the UI appears incorrectly as a result of the image modification.